### PR TITLE
docs: update readme to acknowledge Angular 6

### PR DESCRIPTION
**Pull request by amitport**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

## Other information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ The packages from this repo are published as scoped packages under [@nguniversal
 - [@nguniversal/module-map-ngfactory-loader](https://github.com/angular/universal/tree/master/modules/module-map-ngfactory-loader) (for handling lazy loaded modules on the server).
 
 
-The 1.x packages work with Angular 4.x while the 5.x packages will work with Angular 5.x. We will match the major version with Angular starting from 5.0 to avoid confusion.
+The 1.x packages work with Angular 4.x while, starting from Angular 5.0, the major version matches Angular's version. That is, 5.x packages will work with Angular 5.x, 6.x will work with Angular 6.x and so forth.
```
Comment (Toxicable) [inline README.md]: I think this section should be removed. 
I think it's intuitive that nguniversal 6.x will work with angular 6.x

Comment (amitport) [reply README.md]: removed edit and rebased (@Toxicable)

Comment (Toxicable) [reply README.md]: Sorry, I meant that section should be removed from the README all together.
But still, you've addressed the issue here, merging.

Comment (amitport) [reply README.md]: removed

**Comment by Toxicable** [comment]
Just needs a rebase

**Pull request merged**